ui_framework/src/components/context_menu/context_menu_panel.js

```diff
@@ -226,20 +226,19 @@ export class KuiContextMenuPanel extends Component {
   };
 
   panelRef = node => {
-    if (node) {
-      this.panel = node;
+    this.panel = node;
+
+    if (this.panel) {
       this.panel.addEventListener('animationend', this.onTransitionComplete);
```

Comment:
Now if we just could remove the event listener again when the previous `this.panel` was an element but the `node` isn't... or even better replace it with the `onAnimationEnd` react prop :thought_balloon: